README.md

```diff
@@ -1,6 +1,6 @@
 :warning: This codebase is still under construction :warning:
 
-[![Python](https://img.shields.io/badge/python-3.7%2B-blue)]() [![Open Pull Requests](https://img.shields.io/github/issues-pr/ArthurConmy/Automatic-Circuit-Discovery.svg)](https://github.com/ArthurConmy/Automatic-Circuit-Discovery/pulls)
+[![Python](https://img.shields.io/badge/python-3.8%2B-blue)]() [![Open Pull Requests](https://img.shields.io/github/issues-pr/ArthurConmy/Automatic-Circuit-Discovery.svg)](https://github.com/ArthurConmy/Automatic-Circuit-Discovery/pulls)
```

Comment:
```
  - transformer-lens requires Python >=3.7.4,<4.0, so it will not be satisfied for Python >=3.7,<3.7.4

  - pygraphviz requires Python >=3.8, so it will not be satisfied for Python >=3.7.4,<3.8
```